.github/workflows/pull-request.yml

```
@@ -230,11 +237,12 @@ jobs:
           done
 
-      - name: Upload Coverage to Codecov
+      - name: Present and Upload coverage to Codecov as ${{env.CODECOV_UNIQUE_NAME}}
         uses: codecov/codecov-action@v4
         with:
           token: ${{ secrets.CODECOV_TOKEN }}
           verbose: true
+          fail_ci_if_error: false
           files: './coverage/lcov.info'
           name: '${{env.CODECOV_UNIQUE_NAME}}'
           fail_ci_if_error: false
```

actionlint (1.7.4)

key "fail_ci_if_error" is duplicated in "with" section. previously defined at line:245,col:11. note that this key is case insensitive

(syntax-check)